ui/components/multichain/detected-token-banner/detected-token-banner.stories.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { Provider } from 'react-redux';
+
+import testData from '../../../../.storybook/test-data';
+import configureStore from '../../../store/store';
+import { DetectedTokensBanner } from './detected-token-banner';
+
+const store = configureStore(testData);
+
+export default {
+  title: 'Components/Multichain/DetectedTokensBanner',
+  component: DetectedTokensBanner,
+  decorators: [(story) => <Provider store={store}>{story()}</Provider>],
+
+  argTypes: {
+    setShowDetectedTokens: { control: 'func' },
+  },
+  args: {
+    setShowDetectedTokens: 'setShowDetectedTokensSpy',
+  },
+};
+
+export const DefaultStory = (args) => <DetectedTokensBanner {...args} />;
+
+DefaultStory.storyName = 'Default';
```

Comment:
Getting a propType error in this story

<img width="1440" alt="Screenshot 2023-03-08 at 3 51 40 PM" src="https://user-images.githubusercontent.com/8112138/223883583-6dbe3fcd-fa68-489b-a42b-82baab84b97c.png">